packages/core/src/Menu/Menu.stories.js

```diff
@@ -22,6 +23,13 @@ export const Dense = () => (
     </Menu>
 )
 
+export const AutoHideFirstSectionHeaderDivider = () => (
+    <Menu>
+        <MenuSectionHeader label="First - no divider above" />
+        <MenuSectionHeader label="Second - with divider above" />
+    </Menu>
+)
+
```

Comment:
I think this should go into the `MenuSectionHeader.stories.js`

Reply:
Yeah I started of having it there. But in the end I moved it, because this is actually a feature of the Menu.

Having said that, I didn't realise that the JSX panel was showing the decorator markup too. I suppose moving it there does make sense after all. I'll do that.

Reply:
Correction. I tried that and because we're using a decorator in that story already it's only possible to render two headers in a fragment, and that breaks the logic, since the menu is expecting its children to be the items. So to change this, I'd have to remove the decorator from the MenuSectionHeader demo and update all the stories there.

It's not that the above would be much work, but to me it just underpins that this is actually a feature of the Menu, and should be demoed as such.

Reply:
Hmm to me, all these different story "namespaces" are quite confusing. I think I'd prefer if everything could be found under `Menu`